### PR TITLE
Feat: Onboard Human Variant Annotation dataset

### PR DESCRIPTION
**Pull request by vijay-google**
## Description

Dataset: human_variant_annotation pipeline: clinvar
Dataset: human_variant_annotation pipeline: db_snp

## Checklist

- [x] **(Required)** This pull request is appropriately labeled
- [x] Please merge this pull request after it's approved
- [x] I'm adding or editing a dataset
- [x] The [Google Cloud Datasets team](mailto:cloud-datasets-onboarding@google.com) is aware of the proposed dataset
- [x] I put all my code inside  `datasets/human_variant_annotation` and nothing outside of that directory

### CODE REVIEW

File: datasets/human_variant_annotation/pipelines/_images/run_csv_transform_kub/csv_transform.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+
+def main(
+    base_url: str,
+    folder: pathlib.Path,
+    version: str,
+    gcs_bucket: str,
+    target_gcs_folder: str,
+    pipeline: str,
+):
```
Comment (nlarge-google) [inline datasets/human_variant_annotation/pipelines/_images/run_csv_transform_kub/csv_transform.py:32]: Always incorporate a return type.  For no return type use:

def fn(...) -> None:

File: datasets/human_variant_annotation/pipelines/_images/run_csv_transform_kub/csv_transform.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    version: str,
+    folder: pathlib.Path,
+    gcs_bucket: str,
+    target_gcs_folder: str,
+):
+    file_name = f"clinvar_{date_time.strftime('%Y%m%d')}.vcf.gz"
+    source_url = base_url + f"archive_{version}/{date_time.strftime('%Y')}/{file_name}"
+    source_file = f"./files/{folder}/{file_name}"
+    status_code = download_gzfile(source_url, source_file)
+    if status_code == 200:
```
Comment (nlarge-google) [inline datasets/human_variant_annotation/pipelines/_images/run_csv_transform_kub/csv_transform.py:74]: else:
     pass

File: datasets/human_variant_annotation/pipelines/_images/run_csv_transform_kub/csv_transform.py [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+
+def get_files(
+    date_time: datetime.datetime,
+    base_url: str,
+    version: str,
+    folder: pathlib.Path,
+    gcs_bucket: str,
+    target_gcs_folder: str,
+):
```
Comment (nlarge-google) [inline datasets/human_variant_annotation/pipelines/_images/run_csv_transform_kub/csv_transform.py:69]: return type None

**Review by nlarge-google — CHANGES_REQUESTED** [review]
Minor changes.  Please make the changes and retest.

File: datasets/human_variant_annotation/pipelines/_images/run_csv_transform_kub/csv_transform.py (hunk 1 above)
Comment (vijay-google) [reply datasets/human_variant_annotation/pipelines/_images/run_csv_transform_kub/csv_transform.py:32]: Ok done

File: datasets/human_variant_annotation/pipelines/_images/run_csv_transform_kub/csv_transform.py (hunk 2 above)
Comment (vijay-google) [reply datasets/human_variant_annotation/pipelines/_images/run_csv_transform_kub/csv_transform.py:74]: ok done

File: datasets/human_variant_annotation/pipelines/_images/run_csv_transform_kub/csv_transform.py (hunk 3 above)
Comment (vijay-google) [reply datasets/human_variant_annotation/pipelines/_images/run_csv_transform_kub/csv_transform.py:69]: ok done

**Review by nlarge-google — APPROVED** [review]
Great Job.  In future though, please reduce the number of blank lines between code blocks to 0.  Thanks!

**Pull request merged**